code/modules/security_levels/keycard_authentication.dm

```diff
@@ -101,6 +101,10 @@ GLOBAL_DATUM_INIT(keycard_events, /datum/events, new)
 	event_source = null
 
 /obj/machinery/keycard_auth/proc/trigger_event(confirmer)
+	if(confirmer == triggerer)
```

Comment:
who the fuck came up with these variable names

Reply:
/shrug